### PR TITLE
Grammar corrections PR#23206 Review Fixes

### PR DESCRIPTION
**Pull request by masayafunakoshi1**
#### Problem
A couple of notes missed when making last PR
#### Summary of Changes
Made edits/fixes where recommended

Fixes # 
Check PR #23206 and comments under it for more specifics

### CODE REVIEW

File: docs/src/developing/clients/jsonrpc-api.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ Unstable methods may see breaking changes in patch releases and may not be suppo
 
 ## Request Formatting
 
-To make a JSON-RPC request, send an HTTP POST request with a `Content-Type: application/json` header. The JSON request data should contain 4 fields:
+To make a JSON-RPC request, send an HTTP POST request with a `Content-Type:
```
Comment (CriesofCarrots) [inline docs/src/developing/clients/jsonrpc-api.md:110]:
```suggestion
To make a JSON-RPC request, send an HTTP POST request with a `Content-Type:
```

File: docs/src/developing/programming-model/accounts.md [hunk 2]
```diff
@@ -23,11 +23,12 @@ uses an _address_ to look up an account. The address is a 256-bit public key.
...
-Transactions may include digital [signatures](terminology.md#signature)
-corresponding to the account's public keys referenced by the transaction. Such
-signatures signify that the holder of the
-account's private key signed, and thus, "authorized" the transaction. In this case,
-the account is referred to as a _signer_. Whether an account is a signer or not
+
+Transactions include one or more digital [signatures](terminology.md#signature)
+each corresponding to an account address referenced by the transaction. Each of these
+addresses must be the public key of an ed25519 keypair, and the signature signifies
+that the holder of the matching private key signed, and thus, "authorized" the transaction.
```
Comment (CriesofCarrots) [inline docs/src/developing/programming-model/accounts.md:30]:
```suggestion
that the holder of the matching private key signed, and thus, "authorized" the transaction.
```

**Review by CriesofCarrots — COMMENTED** [review]
Thank you so much for following up!
Our sanity CI job is complaining about some dangling whitespace, which I've fixed in suggestions. Feel free to just merge those directly.

Then can you please rebase on master?

File: docs/src/developing/programming-model/accounts.md (hunk 2 above)
Comment (masayafunakoshi1) [reply docs/src/developing/programming-model/accounts.md:30]: Oooo gotcha, will do!

**Review by CriesofCarrots — APPROVED** [review]
Thanks!

**Pull request merged**